src/firefly/js/ui/ValidationField.jsx

```diff
@@ -1,13 +1,19 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import {has} from 'lodash';
-
+import {has, get} from 'lodash';
 import {InputFieldView} from './InputFieldView.jsx';
 import {fieldGroupConnector} from './FieldGroupConnector.jsx';
 
 
 function onChange(ev, store, fireValueChange) {
     var value = ev.target.value;
+
+    //html5 validation, using validity property object. If it is marked invalid, then no need to sent it to validator.js
+    if(! get(ev, 'target.validity.valid')) {
+        fireValueChange({value, message: get(ev, 'target.validationMessage','Invalid date entered') , valid:false});
+        return;
```

Comment:
The should be no mention of date here.  `ValidationField` works for many types not just date.